echopype/preprocess/api.py

```diff
@@ -329,3 +284,427 @@ def remove_noise(ds_Sv, ping_num, range_sample_num, noise_max=None, SNR_threshol
 
 def regrid():
     return 1
+
+
+def get_bin_indices(
+    echo_range: np.ndarray, bins_er: np.ndarray, times: np.ndarray, bins_time: np.ndarray
+) -> Tuple[np.ndarray, np.ndarray]:
+    """
+    Obtains the bin index of ``echo_range`` and ``times`` based
+    on the binning ``bins_er`` and ``bins_time``, respectively.
+
+    Parameters
+    ----------
+    echo_range: np.ndarray
+        2D array of echo range values
+    bins_er: np.ndarray
+        1D array (used by np.digitize) representing the binning required for ``echo_range``
```

Comment:
This should also be a 2D array since `echo_range` is 2D?
```suggestion
    bins_er: np.ndarray
        2D array (used by np.digitize) representing the binning required for ``echo_range``
        ``bins_er`` should be of the same shape as ``echo_range``
```

Reply:
As we discussed, `bin_er` should not be a 2D array. I will resolve this conversation.